libs/LedKit/source/animations/Rainbow.cpp

```diff
@@ -78,13 +78,14 @@ void Rainbow::stagesRainbow()
 
 auto Rainbow::getRainbowColor(uint8_t index) const -> RGB
 {
-	static constexpr auto kRainbowColorNumber						   = 7;
-	static constexpr std::array<RGB, kRainbowColorNumber> rainbowColor = {
-		RGB::pure_red,	RGB {180, 50, 0},	 RGB {100, 100, 0}, RGB::pure_green,
-		RGB::pure_blue, RGB {0x4b, 0, 0x82}, RGB::magenta};
-
-	auto part_of_rainbow = index / 3;
-	auto position		 = static_cast<float>(index % 3) / 3;
+	static constexpr auto kNumberOfLedsBelt							   = uint8_t {20};
+	static constexpr auto kRainbowColorNumber						   = 5;
+	static constexpr std::array<RGB, kRainbowColorNumber> rainbowColor = {RGB::pure_red, RGB::pure_green,
+																		  RGB::pure_blue, RGB::magenta, RGB::pure_red};
```

Comment:
en vrai pas besoin de `kRainbowColorNumber`, il suffit d'utiliser la taille du std::array, c'est quand même plus pratique

```suggestion
	static constexpr auto colors = std::to_array<RGB>({
		RGB::pure_red,
		RGB::pure_green,
		RGB::pure_blue,
		RGB::magenta,
		RGB::pure_red,
	});
```